File: app/src/main/java/com/mundi/databaseadmin/bean/RowTitle.java
package com.mundi.databaseadmin.bean;

public class RowTitle {

    private String str;
    private boolean flag = false;

    public RowTitle(int index) {
        this.str = Integer.toString(index);
        this.flag = true;
    }

    public RowTitle(String str) {
        this.str = str;
        this.flag = true;
    }

    public String getRowTitle() {
        return str;
    }

    public void setFlag(boolean flag) {
        this.flag = flag;
    }

    public boolean getFlag() {
        return flag;
    }

}

File: app-kotlin/src/main/java/com/mundi/databaseadmin/bean/RowTitle.kt
package com.mundi.databaseadmin.bean

class RowTitle {
    var rowTitle: String
        private set
    var flag = false

    constructor(index: Int) {
        rowTitle = Integer.toString(index)
        flag = true
    }

    constructor(str: String) {
        rowTitle = str
        flag = true
    }

}